lib/openstax/aws/deployment_base.rb

```diff
@@ -11,6 +11,10 @@ def stack_statuses(reload: false)
         end
       end
 
+      def failed?(reload: false)
+        @deployment.stacks.any?(&:failed?)
```

Comment:
need to pass `reload` to `stacks`

Reply:
do I? https://github.com/openstax/aws-ruby/blob/4a929a63708125733d3e1f86503620b68ddc1fe0/lib/openstax/aws/deployment_base.rb#L175

Reply:
oh this is supposed to be `@deployment.stack_statuses` not `@deployment.stack`.  As is works because the latter delegates to the former via (`delegate :failed?, :updating?, :creating?, :deleting?, :exists?, to: :status`), but if we use the former we can pass in `reload`

Reply:
`stack_statuses(reload: reload).any?(&:failed?)` didn't work for me, because I would need to access &.status:failed? and didn't find how to do that